packages/site-demo/content/product/components/notification/index.mdx

```diff
@@ -1,9 +1,37 @@
 # Notification
 
-## Default
+**Notifications provide information at the bottom of the user's window.**
 
 <DemoBlock demo="default" withThemeSwitcher />
 
-### Properties
+## Sizes
+
+Notifications comes in three sizes: _tiny_, _regular_ or _big_.
+
+### Tiny
+
+Tiny notifications comes in three variants: confirmation, warning, alert or neutral.
```

Comment:
Wrong variant names, in code and parameters:
Type of notification (info, success, warning, error).